client/src/bindings/calendar.js

```diff
@@ -3,19 +3,23 @@ import Pikaday from 'pikaday';
 
 ko.bindingHandlers.calendar = {
     init(element, valueAccessor) {
-        element.value = valueAccessor().value();
+        let {from, to, value} = valueAccessor();
+        [from, to, value] = [from, to, value].map(item => ko.unwrap(item));
```

Comment:
Here you are cautiously unwrapping values, but below you:
1. Use `valueAccessor` again to get reference to `value`;
2. Set its value like it's an observable.

Very inconsistent.